provider/numa/numa_base.py

```diff
@@ -83,7 +80,10 @@ def teardown(self):
             backup_vmxml.sync()
         hpc_2M = self.params.get('hp_config_2M')
         hpc_1G = self.params.get('hp_config_1G')
-        for hpc in [hpc_2M, hpc_1G]:
+        hpc_list = self.params.get('hpc_list', [])
+        hpc_list.append(hpc_2M)
+        hpc_list.append(hpc_1G)
```

Comment:
Would you consider this?
```suggestion
        hpc_list.extend([hpc_2M, hpc_1G])
```

Reply:
Updated